src/oop/oop1.py

```diff
@@ -17,3 +17,33 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+
+# Base class is Vehicle:
```

Comment:
Great job on getting all the tests to pass, @AnniqueN! :) 
<img width="781" alt="Screen Shot 2019-12-09 at 2 01 44 AM" src="https://user-images.githubusercontent.com/16110931/70414409-fd10ba00-1a27-11ea-8bf1-767bfab009a7.png">